src/thunderbots/software/ai/world/world.cpp

```diff
@@ -115,7 +118,18 @@ Team &World::mutableEnemyTeam()
 
 void World::updateRefboxGameState(const RefboxGameState &game_state)
 {
-    game_state_.updateRefboxGameState(game_state, ball_);
+    refbox_game_state_history.push_back(game_state);
+    // Take the consensus of the past 3 refbox messages
+    if (refbox_game_state_history.size() >= 3 &&
+        refbox_game_state_history[0] == refbox_game_state_history[1] &&
+        refbox_game_state_history[0] == refbox_game_state_history[2])
+    {
```

Comment:
Why not take the consensus of the last `n` refbox messages, where `n` is the array length?

Reply:
done